ykman-gui/qml/PivPinManagementView.qml

```diff
@@ -0,0 +1,164 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.2
+import QtQuick.Controls.Material 2.2
+
+ColumnLayout {
+    property bool hasPin
+    property string pinMessage
+    property bool isBusy
+
+    readonly property var pivData: yubiKey.piv || {}
+    readonly property bool pinBlocked: pinRetries < 1
+    readonly property int pinRetries: pivData.pin_tries
+    readonly property bool pukBlocked: yubiKey.pivPukBlocked
+
+    StackView.onActivating: load()
+
+    objectName: "pivPinManagementView"
+
+    function load() {
+        isBusy = true
+        yubiKey.refreshPiv(function () {
+            isBusy = false
+        })
+    }
+
+    function getPinMessage() {
+        if (pinBlocked) {
+            return qsTr("PIN is blocked.")
+        } else {
+            return qsTr("%1 PIN retries left").arg(pinRetries)
+        }
+    }
+
+    function getPukMessage() {
+        if (pukBlocked) {
+            return qsTr("PUK is blocked.")
+        } else {
+            return ""
+        }
+    }
+
+    BusyIndicator {
+        Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
+        running: isBusy
+        visible: running
+    }
+
+    CustomContentColumn {
+        visible: !isBusy
+
+        ColumnLayout {
+            Layout.alignment: Qt.AlignLeft | Qt.AlignTop
+            Heading1 {
+                text: qsTr("Configure PIV PINs")
```

Comment:
Let's use "Configure PINs" as a heading, thats what the button and breadcrumb say.

Reply:
Ok!

Reply:
Fixed!